server/app.ts

```diff
@@ -16,6 +16,7 @@ import setUpValidationMiddleware from './middleware/setUpValidationMiddleware'
 import indexRoutes from './routes'
 import productRoutes from './routes/products'
 import componentRoutes from './routes/components'
+import componentEnvRoutes from './routes/componentsEnvDetails'
```

Comment:
Personally I'd expand `env` or `Env` to environment so it is clear.

Reply:
Updated